Fix: Firebase - Env Variable groups (#517)
codemagic-ci-cd/codemagic-docs: content/yaml-testing/firebase-test-lab.md

## Configure testing in Firebase Test Lab in codemagic.yaml

> To access the Firebase project from Codemagic, add the service account JSON key file and the Firebase project name as environment variables in the **Environment variables** section in Codemagic UI (either in Application/Team variables). Select **secure** to encrypt the values. 

Reviewer: ```suggestion
To access the Firebase project from Codemagic, add the service account JSON key file and the Firebase project name as environment variables in the **Environment variables** section in Codemagic UI (either in Application or Team variables). Save the variables to the same group. Select **secure** to encrypt the values. 
```

> Add the group for the environment variable to `codemagic.yaml` as belows:

Reviewer: ```suggestion
Then import the group to `codemagic.yaml` this way:
```

```yaml
environment:
  groups:
    - firebase_credentials # <-- (Includes: GCLOUD_KEY_FILE - service account JSON key file)
  vars:

>     FIREBASE_PROJECT: "YOUR_FIREBASE_PROJECT_NAME" # <-- Put your Firebase Project Name here

Reviewer: the vars section seems unnecessary

Author: But this can be added 
FIREBASE_PROJECT: "YOUR_FIREBASE_PROJECT_NAME" # <-- Put your Firebase Project Name here

Reviewer: We're recommending saving it to environment variables in the UI at the top of the section, hence my comment.

Author: Changes done
